packages/core/test/acceptance/di_spec.ts

```diff
@@ -1919,6 +1919,128 @@ describe('di', () => {
              });
         });
       });
+
+      onlyInIvy('Ivy support InjectFlags in Injector')
```

Comment:
I think you can create a new `describe` section and put all tests inside:
```suggestion
      onlyInIvy('Ivy supports `InjectFlags` in NodeInjector').describe('`InjectFlags` support in NodeInjector', () => {
        // ... other tests go here ...
        // it('...', () => { ... });
      });
```